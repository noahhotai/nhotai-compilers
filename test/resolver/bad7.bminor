

func1: function integer (x:integer, y:char) =
{
    if (1){
        return z;
    }
        
}