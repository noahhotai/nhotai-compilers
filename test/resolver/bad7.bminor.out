Program parsed successfully.
resolve error: z is not defined
Program failed to resolve.
